crates/rolldown/src/bundler.rs

```diff
@@ -108,20 +109,13 @@ impl Bundler {
     {
       Ok(v) => v,
       Err(errs) => {
-        let errors = Arc::new(errs.into_vec());
+        let errors = UniqueArc::new(errs.into_vec());
         self
           .plugin_driver
-          .build_end(Some(&HookBuildEndArgs {
-            errors: Arc::clone(&errors),
-            cwd: &self.options.cwd,
-          }))
+          .build_end(Some(&HookBuildEndArgs { errors: errors.weak_ref(), cwd: &self.options.cwd }))
```

Comment:
Other thing is the rollup only have one error because the js `Error` mechanism，the rolldown `error` is mutilply, we should merge it to one error.